mergeable/src/Mergeable.ts

```diff
@@ -0,0 +1,28 @@
+export async function checkMergeability(slack: Slack, gitHub: GitHub, secrets: Secrets): Promise<any> {
+    const result: Result = await fetchGitHubMergeability(gitHub, secrets.gitHub)
```

Comment:
you don't need to specify the type since that's the return type of `fetchGitHubMergeability`